### DISCUSSION TITLE
Show Map on shop page

### DISCUSSION THREAD

**Discussion by Ahrdie** [pretix/pretix#2524] [Q&A] [answered]
Hi, I want to integrate a map of the event location on the shop page.
I already configured the API keys, so that I get a map in the shop configuration page ([like this](https://github.com/pretix/pretix/pull/1458)).

Is it possible, to shop the event location on a leaflet map on the public shop front page?

**Comment by raphaelm** [comment, marked as answer]
No, currently not

**Comment by raphaelm** [comment]
(duplicate of #1939)
